test/integration/vaults/vault-creation.integration.ts

```diff
@@ -0,0 +1,250 @@
+    expect(await delegation.connect(owner).withdrawableEther()).to.equal(2n);
+    expect(await delegation.connect(withdrawer).withdraw(stranger, 2n)).to.be.ok;
+    expect(await delegation.connect(owner).withdrawableEther()).to.equal(0);
+  });
+
+  it("Allows to pause/resume deposits to validators", async () => {
+    expect(await delegation.connect(depositPausers).pauseBeaconChainDeposits()).to.be.ok;
+    expect(await delegation.connect(depositResumers).resumeBeaconChainDeposits()).to.be.ok;
+  });
+
+  it("Allows to ask Node Operator to withdraw funds from validator(s)", async () => {
+    const vaultOwnerAddress = await stakingVault.owner();
+    const vaultOwner: ContractRunner = await impersonate(vaultOwnerAddress, ether("10000"));
+    expect(await stakingVault.connect(vaultOwner).requestValidatorExit(SAMPLE_PUBKEY)).to.be.ok;
+  });
+
+  it("Allows to trigger validator withdrawal", async () => {
+    const vaultOwnerAddress = await stakingVault.owner();
+    const vaultOwner: ContractRunner = await impersonate(vaultOwnerAddress, ether("10000"));
+
+    expect(
+      await stakingVault
+        .connect(vaultOwner)
+        .triggerValidatorWithdrawal(SAMPLE_PUBKEY, [ether("1")], vaultOwnerAddress, { value: 1n }),
+    ).to.be.ok;
+  });
+
+  describe("Allows actions only after connecting to Hub", () => {
+    it("Allows to mint stEth", async () => {
+      const { vaultHub } = ctx.contracts;
+
+      await expect(delegation.connect(minter).mintStETH(stranger, 1n)).to.be.revertedWithCustomError(
+        vaultHub,
+        "NotConnectedToHub",
+      );
+      await connectToHub();
+
+      await expect(delegation.connect(minter).mintStETH(stranger, 1n)).to.not.be.revertedWithCustomError(
+        vaultHub,
+        "NotConnectedToHub",
+      );
+      await disconnectFromHub();
+    });
+
+    it("to burn stEth", async () => {
+      const { vaultHub } = ctx.contracts;
+
+      // todo: why it fails with ALLOWANCE_EXCEEDED before NotConnectedToHub? How to fix it?
+      await expect(delegation.connect(burner).burnStETH(1n)).to.be.revertedWithCustomError(
+        vaultHub,
+        "NotConnectedToHub",
+      );
+      await connectToHub();
+
+      await expect(delegation.connect(burner).burnStETH(1n)).to.not.be.revertedWithCustomError(
+        vaultHub,
+        "NotConnectedToHub",
+      );
+      await disconnectFromHub();
+    });
+
+    describe("claiming fees", async () => {
+      before(async () => {
+        await delegation.connect(funder).fund({ value: ether("1") });
+        await generateFeesToClaim();
+      });
+
+      it("to claim Curator's fee", async () => {
+        const { vaultHub } = ctx.contracts;
+        //await connectToHub();
+        // todo: expecting for method to be reverted because we are not connected to the hub, but it is passing fine
+        await expect(delegation.connect(curatorFeeClaimers).claimCuratorFee(stranger)).to.be.revertedWithCustomError(
```

Comment:
Dima says that `claimCuratorFee` and `claimNodeOperatorFee` should not work until connected to hub, but here they pass fine

And if I manually disconnect from Hub or don't do `await stakingVault.connect(hubSigner).report(rewards, 0n, 0n);`,  it gives ZeroArgument

Does it work as it is supposed to work?